ecosystem/nft-metadata-crawler/nft-metadata-crawler-parser/src/parser_entry.rs

```diff
@@ -168,21 +174,67 @@ impl ParserEntry {
     }
 
     /// Calls and handles error for writing JSON to GCS
-    async fn handle_write_json_to_gcs(&mut self, _json: Option<Value>) -> Option<String> {
-        todo!();
+    async fn handle_write_json_to_gcs(&mut self, json: Option<Value>) -> Option<String> {
```

Comment:
If when json is null, you'll log an error, then just log the error before getting into this function.